geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/SqlHandler.java

```diff
@@ -54,6 +60,13 @@ public SqlHandler(InternalCache cache, String regionName,
     this.tableMetaDataManager = tableMetaDataManager;
     this.regionMapping = getMappingForRegion(configService, regionName);
     this.dataSource = getDataSource(dataSourceFactory, this.regionMapping.getDataSourceName());
+    try {
+      compareTableMetadataWithMapping(tableMetaDataManager, regionMapping, regionName);
+    } catch (Exception ex) {
+      System.out.println("Throwing Exception...");
```

Comment:
Don't put System.out.println in the code. They are okay for debugging but should be removed before review

Reply:
These are still in the PR since we have a bit more unit test work to do. This output is very helpful for debugging test code for this story, but after we finish those tests it will be removed.